### ISSUE TITLE
Research drive for storage of challenge data

### ISSUE THREAD

**Issue by Maartenvm** [EYRA-Benchmark/grand-challenge.org#2]
Make a nice interface to upload (and link) public challenge data like training sets, sample submissions etc.
Allow for upload of private data to enable test set / ground truth / validation set upload etc.
Integrate into Research drive.
